### PR TITLE
Add EIP: Leaf-level state expiry in verkle trees

### PR DESCRIPTION
**Pull request by gballet**
This proposal relies on the structure of verkle trees to implement state expiry. A counter is maintained at the extension node level, and only extention-and-suffix nodes (colloquially referred to as "leaves") are deleted.

This is based on the EPF work done by @weiihann, with some added verkle-friendliness.

### CODE REVIEW

File: EIPS/eip-template.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
+---
```
Comment (abcoathup) [inline EIPS/eip-template.md:1]:
```suggestion
---
eip: 7736
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

File: EIPS/eip-template.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+title: Leaf-level state expiry in verkle trees
+description: Simple state expiry scheme in which only "extension-and-suffix trees" are expired.
+author: Guillaume Ballet (@gballet), Wei Han Ng (weiihann)
```
Comment (abcoathup) [inline EIPS/eip-template.md:4]:
```suggestion
author: Guillaume Ballet (@gballet), Wei Han Ng (@weiihann)
```
Assume you meant to add https://github.com/weiihann as an author.

File: EIPS/eip-template.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+discussions-to: https://ethereum-magicians.org/t/leaf-level-state-expiry-in-verkle-trees/20474/1
```
Comment (abcoathup) [inline EIPS/eip-template.md:5]:
```suggestion
discussions-to: https://ethereum-magicians.org/t/eip-7736-leaf-level-state-expiry-in-verkle-trees/20474
```
Renamed Eth Magicians topic with EIP number and linked to topic

File: EIPS/eip-template.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+The resurrection transaction is defined as follows:
+
+`RESURRECT_TX_TYPE|ssz(Vector[stem,last_epoch,values])`
+
+Where:
+
+ * `stem` is used to find the location in the tree, so that the node can be recreated;
+ * `last_epoch` and `values` are the items that were deleted;
+
+At the start of the validation, charge the costs using constants defined in [EIP-4672](./eip-4762.md):
```
Comment (abcoathup) [inline EIPS/eip-template.md:99]:
```suggestion
At the start of the validation, charge the costs using constants defined in [EIP-4762](./eip-4762.md):
```
Assume you mean EIP-4762: Statelessness gas cost changes

**Review by g11tech — APPROVED** [review]

**Pull request merged**